### PR TITLE
refactor(hours request): add scholarship request endpoints

### PR DESCRIPTION
**Pull request by JakecorralesCespedes**

### CODE REVIEW

File: src/scholarship-request/dto/create-scholarship-request.dto.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    @ApiProperty()
+    @IsNumber()
+    departmentId: number;
+    @ApiProperty()
+    @IsNumber()
+    studentId: number;
+    @ApiProperty()
+    status: RequestStatus;
+}
+``
```
Comment (wmartzh) [inline src/scholarship-request/dto/create-scholarship-request.dto.ts:14]: Remove this

**Review by wmartzh — CHANGES_REQUESTED** [review]
Please make sure to run the `npm run format`

File: src/scholarship-request/scholarship-request.service.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+} from '../utils/pagination.util';
+
+@Injectable()
+export class ScholarshipRequestService {
+  constructor(private readonly prismaService: PrismaService) {}
+  async createScholarshipRequest(
+    data: CreateScholarshipRequestDto,
+  ): Promise<StudentOnDepartment> {
+    try {
+      return this.prismaService.studentOnDepartment.create({
```
Comment (wmartzh) [inline src/scholarship-request/scholarship-request.service.ts:20]: Please validate that the student and department exist before create

File: src/scholarship-request/dto/create-scholarship-request.dto.ts (hunk 1 above)
Comment (JakecorralesCespedes) [reply src/scholarship-request/dto/create-scholarship-request.dto.ts]: line 14?? or from line 11 to 14??

**Pull request closed without merging**